xbmc/FileItem.cpp

```diff
@@ -3334,7 +3334,7 @@ std::string CFileItem::GetThumbHideIfUnwatched(const CFileItem* item) const
                                     CSettings::VIDEOLIBRARY_THUMB_SHOW_UNWATCHED_EPISODE) &&
       item->HasArt("thumb"))
   {
-    const std::string fanArt = item->GetArt("fanart");
+    std::string fanArt = item->GetArt("fanart");
```

Comment:
why this change?

Reply:
nvm, found it.

Reply:
If this variable is const, it cannot be moved at the return statement.

see https://clang.llvm.org/extra/clang-tidy/checks/performance-no-automatic-move.html & https://github.com/xbmc/xbmc/pull/19882#discussion_r652444778